docs/developer/extending/apps/asynchronous-webhooks.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: Asynchronous webhooks
+---
+
+## Architecture
+
+Asynchronous webhooks can be used for reacting to the events emitted by the Saleor. Worker instance will send event payload via the chosen protocol. Your app should acknowledge that payload was received. The result of app operations is not sent back to the Saleor. If you need to send data back to the user (for example, calculated shipping cost), you should use [synchronous webhook](developer/extending/apps/synchronous-webhooks.mdx).
+
+## Example use cases
+
+- send order confirmation to company chat.
+- synchronize stocks with an external warehousing system.
+- update Elasticsearch or Algolia index after product description change.
+- trigger frontend SSR for new pages.
+
+## Permissions
+
+Managing webhooks is available for users with `MANAGE_APPS` permission. The app can also create and modify its own webhooks without the need for additional permissions.
+
+## Webhook protocols
+
+Webhooks support below protocols:
+
+- HTTP(S) 
+- Google Cloud Pub/Sub
+- AWS SQS
+
+Protocol is chosen based on `targetUrl` scheme.
+
+### HTTP(S)
+
+Webhook payloads are sent in POST requests.
+
+All webhooks with `targetUrl`, where the scheme is HTTP(S) will use this protocol.
+
+Saleor calculates the HMAC sha256 header - `X-Saleor-Signature` based on `secretKey` and the payload.
+
+If success response is not received, it will retry its request after a delay time in case of temporary service unavailability. 
+
+
```

Comment:
```suggestion
While HTTPS webhooks are a familiar concept and may seem easy to implement, make sure your endpoint can handle the same level of concurrency that you expect from the monitored events. Saleor Cloud will do its best to deliver webhooks in real-time, which could mean more traffic than your servers can handle. Because of this and the possibility of losing events when the destination server is down, we recommend using a queue like Google Cloud Pub/Sub or AWS SQS (see below).

```